.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,87 @@
+name: Build and publish Docker image with version
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    branches: [ master ]
+    tags:
+      - '*.*.*'
+  workflow_dispatch:
+
+jobs:
+  build_and_publish_docker:
+    name: Push Docker image to docker hub and github packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.2.0
+        with:
+          fetch-depth: 0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
+      - name: 'Get Previous tag'
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+      - name: Prepare
+        id: prepare
+        run: |
+          function test_version() {
+            curl -s -S "https://registry.hub.docker.com/v2/repositories/openrouteservice/openrouteservice/tags/?page_size=1024" |
+              sed -e 's/,/,\n/g' -e 's/\[/\[\n/g' |
+              grep '"name"' |
+              awk -F\" '{print $4;}' |
+              sort -fu
+          }
```

Comment:
A short comment on what exactly this one does would be great 😅
I think it shuld return a sorted list of the current tags (latest on top or below?)
But when i run this, it just returns 'next'. Is that intended?

Reply:
this seems to work fine on my machine and in the action's executions.